apps/davi/src/index.tsx

```diff
@@ -43,7 +44,7 @@ const Root = () => {
   }, []);
 
   return (
-    <ApolloProvider client={apolloClient[chainId]}>
+    <ApolloProvider client={apolloClient[chainId][SUPPORTED_SUBGRAPHS.Guilds]}>

```

Comment:
It seems that Apollo needs a client for the context. I tried passing `null` but it crashes.

Passing the Guilds subgraph seems weird, but I don't see another way.

Maybe in the future, this will be replaced with the GuildRegistry subgraph?

Just leaving the comment for future reference.

Reply:
yes, it needs a default client and afterwards on the query you can send the client that you want